Jenkinsfile-release

```diff
@@ -6,7 +6,10 @@ pipeline {
     }
 
     parameters {
-        string(defaultValue: '0.0.0.Qualifier', description: 'Release version', name: 'version')
+        string(name: 'version', defaultValue: '0.0.0.Qualifier', description: 'Release version')
+        string(name: 'nextVersion', defaultValue: '', description: 'Next releasae (blank to stay on current SNAPSHOT)')
```

Comment:
Typo: releasae

Reply:
I'll remove the Jenkinsfile-release as it doesn't work properly still.